src/assemble_workflow/dists.py

```diff
@@ -26,11 +27,12 @@ class Dists:
     }
 
     @classmethod
-    def create_dist(cls, name: str, path: str, min_path: str, distribution: str) -> Dist:
+    def create_dist(cls, name: str, path: str, min_path: str, build: BuildManifest.Build) -> Dist:
+        distribution = build.distribution
```

Comment:
distribution = build.distribution or 'tar'

I don't think you need that info log.

Reply:
sure.

Reply:
Also fixed now.